### ISSUE TITLE
Problem with the authentification

### ISSUE THREAD

**Issue by sipe-daniel** [MitchelSt/react-testing-starter#2]
After running **yarn install && yarn dev** I inserted a password s3cret with the username Tavares_Barrows and I have had a response: Username or password is invalid

I use ubuntu 20.04

**Comment by johnofisher01** [comment]
I too am struggling with authetication with johndoe. 

    {
      "id": "t45AiwidW",
      "uuid": "6383f84e-b511-44c5-a835-3ece1d781fa8",
      "firstName": "John",
      "lastName": "Doe",
      "username": "johndoe",
      "password": "$2a$10$5PXHGtcsckWtAprT5/JmluhR13f16BL8SIGhvAKNP.Dhxkt69FfzW",
      "email": "john@doe.com",
      "phoneNumber": "625-316-9882",
      "avatar": "https://cypress-realworld-app-svgs.s3.amazonaws.com/t45AiwidW.svg",
      "defaultPrivacyLevel": "public",
      "balance": 9950000,
      "createdAt": "2019-08-27T23:47:05.637Z",
      "modifiedAt": "2020-05-21T11:02:22.857Z"
    },


i even tried these to access the app but no luck

**Comment by phut-tran** [comment]
I encountered the same problem. You can run `yarn list:dev:users` to see a list of all users. Check out [original repo](https://github.com/cypress-io/cypress-realworld-app#getting-started).

**Comment by temitopetoluwalase** [comment]
Nah, don't use `yarn dev` instead use `yarn start`. Solves it for me
